package.json

```diff
@@ -19,11 +20,18 @@
   "homepage": "https://github.com/feefs/discordbot#readme",
   "dependencies": {
     "discord.js": "^12.2.0",
+    "discord.js-commando": "github:discordjs/Commando",
```

Comment:
Is there no suitable version on npm?

Reply:
nope. this is the version that is compatible with discord.js v12

Reply:
![:pepehands:](https://user-images.githubusercontent.com/65937586/82988432-011c9e00-9fae-11ea-9189-31c0568e73cb.png)